### PR TITLE
Refactor F# template tests

### PR DESCRIPTION
**Pull request by rwx788**
Besides reusing same structure for the template tests and update of the expectations. I've also moved tests to the "com.jetbrains.rider.plugins.fsharp.test.cases" to have same structure as we have in the main repo.

Changes to the build.gradle.kts were required to run template tests in the correct order, as otherwise we get unpredictable results when trying to test project model before the build.

Requires https://jetbrains.team/p/net/reviews/9605/files (will be merged together).

See [RIDER-85638](https://youtrack.jetbrains.com/issue/RIDER-85638/Refactor-tests-in-F-plugin).

### CODE REVIEW

File: rider-fsharp/testData/templates/net60/Net60/XUnitProjectTemplateTest/projectModel/gold/projectModel_win.gold [hunk 1]
```diff
@@ -29,7 +29,7 @@ Project [Name:Miscellaneous Files, GUID:GUID]
 
 Project [Name:UnitTestProject, GUID:GUID]
   Child items:
-    ProjectFileImpl(Path : Microsoft.TestPlatform.PlatformAbstractions.dll (../../../../../../../.nuget/packages/microsoft.testplatform.testhost/17.1.0/build/netcoreapp2.1/x64/Microsoft.TestPlatform.PlatformAbstractions.dll))[Hidden CONTENT ]
+    ProjectFileImpl(Path : Microsoft.TestPlatform.PlatformAbstractions.dll (../../../113942792/dotnetHomeCli/.nuget/packages/microsoft.testplatform.testhost/17.1.0/build/netcoreapp2.1/x64/Microsoft.TestPlatform.PlatformAbstractions.dll))[Hidden CONTENT ]
```
Comment (auduchinok) [inline rider-fsharp/testData/templates/net60/Net60/XUnitProjectTemplateTest/projectModel/gold/projectModel_win.gold:32]: Is it safe to commit the `113942792` string? It looks like something that can change and/or be different on different machines.

**Review by auduchinok — APPROVED** [review]
@rwx788 Thanks a lot for cleaning all this up!

**Comment by rwx788** [comment]
@auduchinok I'll merge this one with TeamCity, as we need some changes in the Rider repo too, so both are merged simultaneously. Thank you for your help!

**Comment by rwx788** [comment]
Merged using TeamCity. Cheers!

**Comment by auduchinok** [comment]
I've rebased to make the history linear. Thanks again!

**Pull request closed without merging**